types.cc

```diff
@@ -2023,6 +2023,9 @@ data_value deserialize_aux(const tuple_type_impl& t, View v) {
 
 template<FragmentedView View>
 utils::multiprecision_int deserialize_value(const varint_type_impl&, View v) {
+    if (v.current_fragment().empty()) {
+        v.remove_current();
+    }
```

Comment:
Maybe it's better to linearize the whole thing? It will end up linearized in memory as part of multiprecision_int, and it's unlikely to be very large.

Reply:
> Maybe it's better to linearize the whole thing?

It would work too. I don't think it matters which solution we choose.